File: paper/src/main/java/com/badbones69/crazyclaims/paper/api/CrazyManager.java
package com.badbones69.crazyclaims.paper.api;

import com.badbones69.crazyclaims.paper.CrazyClaims;
import com.google.inject.Inject;
import com.google.inject.Singleton;

@Singleton
public class CrazyManager {

    private final CrazyClaims plugin;

    @Inject
    public CrazyManager(CrazyClaims plugin) {
        this.plugin = plugin;
    }

    public void load() {
        plugin.getLogger().info("Guten Tag!");
    }

    public void stop() {
        plugin.getLogger().info("Good Bye!");
    }
}

File: paper/src/main/kotlin/com/badbones69/crazyclaims/paper/api/CrazyManager.kt
package com.badbones69.crazyclaims.paper.api

import com.badbones69.crazyclaims.paper.CrazyClaims

class CrazyManager constructor(plugin: CrazyClaims) {

    private var plugin: CrazyClaims? = null

    init {
        this.plugin = plugin
    }

    fun start() {
        plugin?.logger?.info("Guten Tag!")
    }

    fun stop() {
        plugin?.logger?.info("Good Bye!")
    }
}